client/src/components/PhotoGallery.jsx

```diff
@@ -1,27 +1,72 @@
 import React from 'react';
 import styled from 'styled-components';
+import PhotoGalleryModal from './PhotoGalleryModal.jsx';
 
 const StyledPhotoGallery = styled.div`
   grid-row: 2;
   grid-column: 6 / -1;
-  border-style: solid;
+
+  margin-bottom: 1100px;
 `;
 
-class PhotoGallery extends React.Component {
-  constructor(props) {
-    super(props);
+const StyledSeeAllPhotos = styled.span`
+  color: #0E70BE;
+  :hover {
+    cursor: pointer;
+  }
+  text-decoration: underline;
+  font-size: 12px;
+  float: right;
+  grid-row: 1;
+  grid-column: 1;
+  text-align: right;
+`;
+
+const StyledCustomerPhotosTitle = styled.span`
+  margin-bottom: 0px;
+  margin-top: 32px;
+  font-weight: bold;
+  font-size: 14px;
+  grid-row: 1;
+  grid-column: 1;
+`;
 
-    // this.state = {
+const StyledPhotoGrid = styled.div`
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: 1fr 1fr;
+  grid-row: 2;
+  margin-top: 10px;
+  margin-bottom: auto;
+`;
 
-    // }
+const StyledImage = styled.img`
+  width: 60px;
+  height: 60px;
+  align-content: center;
+  margin-bottom: 5px;
+  margin-top: 5px;
+  margin-left: 5px;
+  margin-right: 5px;
+`;
 
-  }
+const PhotoGalleryHeader = styled.div`
+  margin-top: 32px;
+  max-height: 100px;
+`;
 
-  render() {
-    return (
-      <StyledPhotoGallery>PhotoGallery</StyledPhotoGallery>
-    );
-  };
-};
+const PhotoGallery = (props) => (
+  <StyledPhotoGallery>
+    <PhotoGalleryHeader>
+      <StyledCustomerPhotosTitle>Customer Photos</StyledCustomerPhotosTitle>
+      <StyledSeeAllPhotos onClick={props.showModal}>See All Photos</StyledSeeAllPhotos>
+    </PhotoGalleryHeader>
+    <StyledPhotoGrid>
+      {props.reviews.slice(0, 8).map((review) => (
+        <StyledImage src={review.imageUrl} alt='failed to load' />
```

Comment:
Primary purpose of the alt tag is to provide descriptions of images for those with visual impairments, just FYI! So in this case, every image description would say 'failed to load', not the best for accessibility.